index.html

```diff
@@ -0,0 +1,260 @@
+                            </a>
+                          
+                        </div>
+                        <div class="col-8 header-btn">
+                            <div class="person-icon-acc">
+                                <a href="#" class="p-icon">
+
+                                    <i class="fa fa-fw fa-user" style="background-color: transparent; margin-right: -10px;"></i>
```

Comment:
- [ ] The tablet screen size should say login, not the user icon. The user icon comes on the mobile screen size.

Your clone 👇🏾 
<img width="776" alt="Screen Shot 2021-01-25 at 10 38 29 PM" src="https://user-images.githubusercontent.com/48804258/105769397-0d526400-5f5e-11eb-8210-5b9540a2f271.png">
The original page 👇🏾 
<img width="778" alt="Screen Shot 2021-01-25 at 10 38 47 PM" src="https://user-images.githubusercontent.com/48804258/105769431-180cf900-5f5e-11eb-9210-c54296c31128.png">